docs/source/tutorials/ontologies.rst

```diff
@@ -0,0 +1,26 @@
+********
+Ontologies
+********
+
+A central component of a project is the ontology - an in-depth description of which can be found `here <https://docs.encord.com/ontologies/overview>`_.
+
+The ontology essentially defines the label structure of a given project.
```

Comment:
I'd rather say something like "The ontology defines the template of which the labels can be created". 
Also, "of a given project" might be misleading. Ontologies are top level entities, it is important for people to understand that they're attached to zero or many projects, but ultimately are independent from them

Reply:
Sounds good. I took this bit specifically from our 'Projects' section so will update the text there too for accuracy :)

Reply:
thank you!